lib/tokyo_api/actionkit.rb

```diff
@@ -8,8 +8,12 @@ def full_user(id)
       client.get_request("#{normalized_base_path}full_user/#{id}").body
     end
 
-    def user_path(id, phone_required)
-      "/#{normalized_base_path}user/#{id}?phone_required=#{phone_required}"
+    def user_path(id, required_fields: nil)
```

Comment:
not a new problem but is there a remote url retrieval bug here: should we validate the id so someone is not able to insert tricky IDs that are in turn able to trigger weird HTTP GETs behind our firewall?

Reply:
It probably is not an actually exploitable bug, but maybe worth running the id through a regex to make sure it doesn't have a `://`

Reply:
Will address this in https://www.pivotaltracker.com/n/projects/1037834/stories/150811559